### PR TITLE
Move feeds to feature

### PR DESCRIPTION
**Pull request by ns8482e**
Move feeds to feature

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Lists/Startup.cs [hunk 1]
```diff
@@ -121,4 +107,29 @@ public override void ConfigureServices(IServiceCollection services)
...
+    {
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            // Feeds
+            services.AddScoped<IFeedQueryProvider, ListFeedQuery>();
+
+            services.AddContentPart<ListPart>()
+                .UseDisplayDriver<ListPartFeedDisplayDriver>()
+                .AddHandler<ListPartFeedHandler>();
+
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Lists/Startup.cs:122]: Just remove the two extra lines

**Review by hishamco — APPROVED** [review]
LGTM

**Review by deanmarcussen — CHANGES_REQUESTED** [review]
Good, yes, but I need to check it is non breaking.

Please hold off on merging for a little bit

**Comment by hishamco** [comment]
Sure, I need to try this too ;)

**Comment by deanmarcussen** [comment]
Ok, so it's non breaking which is good.

But I think the comment actually meant to move the `ListPart` feeds provider to an actual feature, so that one could have feeds, without lists being a provider of feeds.

i.e. still providing custom feeds from other sources.

but better change than nothing, so ok with me.

Niraj if you wanted to move it to an actual feature, it would just need to enable the feature by default in a migration on Lists (to be non breaking)

**Comment by ns8482e** [comment]
I changed to hide the following UI for users when `OrchardCore.Feeds` is disabled as it's returning `404`.

image

The change in this PR will only display FeedUrl options when when `OrchardCore.Feeds` is enabled.

**Review by deanmarcussen — APPROVED** [review]

**Pull request merged**